R/options.R

```diff
@@ -30,18 +31,22 @@ load_options <- function(base_path = ".") {
     ))
   }
 
-  utils::modifyList(defaults, opts)
+  c(utils::modifyList(defaults, opts), list(package = descx$package))
 }
 
 load_options_description <- function(base_path = ".") {
   desc_path <- file.path(base_path, "DESCRIPTION")
-  desc_opts <- read.dcf(desc_path, fields = "Roxygen")[[1, 1]]
+  dcf <- read.dcf(desc_path, fields = c("Roxygen", "Package"))
+  desc_opts <- dcf[[1, 1]]
+  desc_pkg <- dcf[[1, 2]]
 
   if (is.na(desc_opts)) {
-    list()
+    opts <- list()
   } else {
-    eval(parse(text = desc_opts), child_env(baseenv()))
+    opts <- eval(parse(text = desc_opts), child_env(baseenv()))
   }
+
+  list(opts = opts, package = desc_pkg)
```

Comment:
This seems a bit too complicated. Could you just inject `package` directly into the returned list here?

Reply:
OK, done.